net/respondd/README.md

```diff
@@ -1,3 +1,109 @@
+# What is respondd?
+
+## Introduction
+respondd is a server distributing information within a network.
+
+For doing so, respondd spawns a udp socket (1001/udp) joining a multicast group. When a request is received, the information requested is transmitted to the requestor. 
+
+All information is oranized in a non-hierachical namespace. Each entry identifies a *provider* implemented in at least one C module. (eg `statistics`, `nodeinfo`, `...`) - each request queries a specific provider. 
+Each response is the merge-result of the output of all modules registered for the provider.
+
+Some providers are implemented in [/package/gluon-respondd](https://github.com/freifunk-gluon/gluon/tree/master).
+## Usage
+
+TODO: Why is the port number included in the multicast group, too?
```

Comment:
The :1001 is not the port, it's just an arbitrary part of the multicast group address.